### ISSUE TITLE
Can place items on wall areas in houses that shouldnt be possible

### ISSUE THREAD

**Issue by Soulless-1** [Sphereserver/Source-X#704]
you can place items here on your walls where you shouldnt be able to , tried to read the terrain and bounce the item with a script fix, but i wasnt able to get the static info from the multi it seems.

![image](https://user-images.githubusercontent.com/17236327/121486131-dc1f8100-c99e-11eb-8422-d00bb8a7c226.png)

**Comment by Tolokio** [comment]
I think this have been like this for many time.
To reach the static from multi u need to do a for among all statics until u get the one with the desired coords.


Does it happens on walls that are not multis?

**Comment by Soulless-1** [comment]
I don't believe this is normal behavior that exists in UO, or that it has existed in the past with sphere, that you can drop items onto a tile that has a wall on it. 

you can't drop item's on walls in the regular game mechanics.

**Comment by drk84** [comment]
closed because duplicate
